### ISSUE TITLE
Documentation For Mac and config.properties

### ISSUE THREAD

**Issue by atilling** [PhoenicisOrg/scripts#691]
Documentation doesn't have the correct location for files on the Mac. The config.properties file isn't in ~/LibraryPhoenicis though most of the user files are there.

I attempt to change my theme and now my Phoenicis crashes when opening, I want to manually update the config to change to the default theme but I can't locate the properties file.

**Comment by atilling** [comment]
I still don't know where Phoenicis was finding the config.properties or Javafx.properties with the bad theme but I created a ~/Library/config.properties with the defaults from Git and I'm working again.

**Comment by plata** [comment]
I'm not on mac here, so I can't really help but it might be useful to check this: https://github.com/PhoenicisOrg/phoenicis/blob/master/phoenicis-configuration/src/main/resources/Mac%20OS%20X.properties

**Comment by atilling** [comment]
When I manually added the ~/Library/config.properties the Mac OS X properties file is where I copied from. Worked great but it seems that there was properties file somewhere that stored the bad theme choice.

**Comment by plata** [comment]
Can you provide a PR with the corrected documentation?

**Comment by atilling** [comment]
I'm still not sure where the config with the bad theme change is/was hiding. Wouldn't be the classpath one because that wouldn't have allowed the change. The ~/Library/phoenix was the fix because it apparently had priority over the bad properties file.

**Comment by madoar** [comment]
What exactly do you mean with bad theme choice?
The currently used theme is saved in `~/.Phoenicis/javafx.properties`.

**Comment by atilling** [comment]
In Phoenicis I attempt to change from the default theme, exited Phoenicis and then restarted. Phoenicis crashed at that point and every time I opened after that. There were no properties files javafx or phoenics in either ~/.Phoenicis/ or in ~/Library/Phoenicis (on the mac ~/.Phoenicis/ only contains the startup log.

The logged output indicated the value of theme was found but not where it was found.

I manually created a phoenics.properties and a javafx.properties using the default values in phoenicis-confguration and phoenicis-javafx and phoenicis is now working for me so it appears that the properties files I created over road the other settings but I assume that there is still a javafx.properties file somewhere on my system with the theme setting that was causing my crash.

**Comment by plata** [comment]
Could this be related to #1418?

**Comment by atilling** [comment]
I don't think so because I was able to initially launch and use Phoenicis it was only after I changed the theme that I couldn't relaunch.

**Comment by plata** [comment]
Ok. So what shall we do about this issue? Just close it?

**Comment by madoar** [comment]
@atilling when you say in `phoenicis-confguration` and `phoenicis-javafx` do you mean inside the source code, inside the jar archive or inside some other folders in `~/.Phoenicis` or `~/Library/Phoenicis`?

I wouldn't yet rule PhoenicisOrg/phoenicis#1418 out, because as far as I understand the spring framework it should save the `javafx.properties` file to `~/.Phoenicis`, which does not occurs on your system, right?

**Comment by atilling** [comment]
I did not have properties files in `~/.Phoenicis` or `~/Library/Phoenicis` I looked at the debug output from the Phoenicis start up and the debug indicated what places the properties files were NOT found - but when it indicated it had found config values it did not give a path for where the properties file WAS found. So I copied the properties file from the source, renamed correctly, and put the file in the first place the debug showed that the file was checked for. `~/Library/Phoenicis` which is also where the documentation says it should be on a Mac.
Once I had the new properties file in place Phoenicis was able to start up again.
